lessons/11_Lesson11_vortexSourcePanelMethod.ipynb

```diff
@@ -537,14 +545,14 @@
     "    \n",
     "    Returns\n",
     "    -------\n",
-    "    Contribution from panel at point (x, y).\n",
+    "    Contribution from the panel at a given point (x, y).\n",
     "    \"\"\"\n",
-    "    def func(s):\n",
+    "    def integrand(s):\n",
     "        return ( ((x - (panel.xa - numpy.sin(panel.beta)*s))*dxdk\n",
```

Comment:
I am aware about this convention, I just do not like it (or do not understand why they recommend it).
I agree that the format of the expressions would be nicer (in our case where the two terms are similar), however, I am just used to read equations written in a similar way as if they were written on paper. (Don't if it makes sense...)